src/envo/envo-edit.owl

```diff
@@ -27234,14 +27234,28 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01000944> <http://purl.obolibrar
 # Class: <http://purl.obolibrary.org/obo/ENVO_01000945> (primary microplastic particle)
 
 AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Microplastics") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://marinedebris.noaa.gov/file/2192/download?token=5dvqb-YY") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01000945> "A microplastic particle which is the direct output of a manufacturing process.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_01000945> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_01000945> <https://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_01000945> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> <http://purl.obolibrary.org/obo/ENVO_01000945> "Primäres Mikroplastik"@de)
```

Comment:
@iwilkie Here and below, I noticed that you added the multilingual interpretations as `has_related_synonym` and not `has_exact_synonym` - was that intentional? At least for the German term I believe it refers to the exact same thing as the English one.